z/histogram.go

```diff
@@ -96,6 +96,11 @@ func (histogram *HistogramData) Update(value int64) {
 	}
 }
 
+// Mean returns the mean value for the histogram.
+func (histogram *HistogramData) Mean() float64 {
+	return float64(histogram.Sum) / float64(histogram.Count)
```

Comment:
No chance of someone calling it when histogram.count is zero? Will it crash or exit gracefully?

Reply:
Nice catch. Fixed it.